examples/tornjak/values.yaml

```diff
@@ -0,0 +1,47 @@
+spire-server:
+  tornjak:
+    enabled: true
+    image:
+      registry: ghcr.io
+      repository: spiffe/tornjak-be
+      pullPolicy: IfNotPresent
+      version: "v1.0.2"
```

Comment:
Probably should leave out most of the default stuff? The chart readme should cover it?

Maybe leave in anything the user is very likely to need to override such as the apiServerURL?

Reply:
oh. and apiServerURL here seems dead. Its the frontend that cares I think?